core/controllers/feedback.py

```diff
@@ -72,7 +72,11 @@ class ThreadHandler(base.BaseHandler):
     @acl_decorators.can_view_feedback_thread
     def get(self, thread_id):
         if constants.USE_NEW_SUGGESTION_FRAMEWORK:
-            suggestion = suggestion_services.get_suggestion_by_id(thread_id)
+            if not constants.ENABLE_GENERALIZED_FEEDBACK_THREADS:
+                suggestion_id = 'exploration.%s' % thread_id
```

Comment:
Do we need to refactor this if we have any other entity in common suggestions view?

Reply:
For other entities, the ENABLE_GENERALISED_FEEDBACK_THREADS flag should be true. in which case, the suggestion_id is just the thread_id, so we wouldn't need a refactor here.